tests/routes/getAllAuthors.test.js

```diff
@@ -45,9 +45,8 @@ describe('Authors Endpoint', () => {
         .get(`${baseRoute}/authors/`)
         .set('Authorization', `Bearer ${userToken}`);
 
-      response.should.have.status(404);
-      response.body.should.be.a('Object');
-      response.body.should.have.property('error');
+      response.should.have.status(200);
+      response.body.data.should.be.a('Array');
```

Comment:
What data are we expecting?

Reply:
Should return an empty array. I just implemented that.